### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jedgarpark**

### CODE REVIEW

File: CPB_Ornament_Proximity/cpb_ornament_proximity.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                 0x000011,
+                 0x110011,
+                 0x111111,
+                 0x221111,
+                 0x112211,
+                 0x111122]
+
+ble = BLERadio()
+
+slide_switch = digitalio.DigitalInOut(board.SLIDE_SWITCH)
```
Comment (ladyada) [inline CPB_Ornament_Proximity/cpb_ornament_proximity.py]: please simplify by using the CircuitPlayground library? you can skip a bunch of this setup :)

Comment (jedgarpark) [reply CPB_Ornament_Proximity/cpb_ornament_proximity.py]: OK, I'll switch things over to CircuitPlayground library.

Comment (ladyada) [reply CPB_Ornament_Proximity/cpb_ornament_proximity.py]: @kattni recently added cpb support and supposedlyu should work just fine :) let us know if it does not

Comment (jedgarpark) [reply CPB_Ornament_Proximity/cpb_ornament_proximity.py]: Got loads of help from @kattni and now it works great using cpb!

**Comment by ladyada** [comment]
ok thanx!

**Pull request merged**